cmd/monogen/testapp/app/app.go

```diff
@@ -167,7 +167,7 @@ func New(
 		AccountKeeper:   app.AccountKeeper,
 		BankKeeper:      app.BankKeeper,
 		SignModeHandler: app.txConfig.SignModeHandler(),
-	})
+	}, app.RollupKeeper)
```

Comment:
Posting for visibility - we'll need to cut a release after this change so the monogen tool can pick up the ante handler updates.

cc @joshklop

Reply:
Monogen is pinned to the `v0.1.1` release, so it isn't a necessity. Am I missing something?

Reply:
Yeah it's not a necessity, however `monogen` will be broken on main when testing the current monomer changes. I think your post in the discord thread made sense and this should be covered in the weekly releases.